examples/psyad/eg2/Makefile

```diff
@@ -40,7 +40,18 @@ CONFIG_ENV = PSYCLONE_CONFIG=${PSYCLONE_DIR}/config/psyclone.cfg
 
 F90FLAGS += -ffree-line-length-none
 
+# The names of 'active' variables in the supplied kernel. Must include those
+# local variables that are deduced to be active because of their dependence
+# on those routine arguments that are active. Note, one of the variables
+# (res_dot_product) did not exist in the original code and is a by-product
+# of psyad replacing a dot_product intrinsic with equivalent code. This and
+# all other local variables will not need to be specified on the command line
+# once issue #1556 has been addressed.
 ACTIVE_VAR_LIST ?= r_u exner theta moist_dyn_gas moist_dyn_tot moist_dyn_fac grad_term theta_v_e theta_v_at_quad grad_theta_v_at_quad exner_e exner_at_quad res_dot_product
+# PSyAD arguments that specify which (if any) of the arguments described in
+# the kernel metadata contain geometric information that must not be modified
+# by the test harness.
+GEOMETRY_VAR_LIST ?=
```

Comment:
Two reasons:

 1. It's not totally specific to eg 2 - it's parameterised so that it can do a range of kernels.
 2. `tl_hydrostatici_kernel` doesn't have any geometry arguments.